src/main/java/com/webonise/todoapp/service/TodoService.java

```diff
@@ -0,0 +1,13 @@
+package com.webonise.todoapp.service;
+
+import java.util.List;
+import org.springframework.http.ResponseEntity;
+import com.webonise.todoapp.model.Todo;
+
+public interface TodoService {
+  
+  public List<Todo> getTodos();
```

Comment:
Try to add documentation above the service methods, Which will state the use of the method.

Reply:
Separate the documentation for each service method.